controllers/games.js

```diff
@@ -54,4 +54,18 @@ exports.postComments = (req, res, next) => {
         .catch((err) => {
             next(err);
         })
+}
+
+exports.patchReviewsById = (req, res, next) => {
```

Comment:
I'm not sure we need a checkExists in this situation, as id is a foreign key I believe we should get an SQL error if a non existent id is passed in to the patch request. Run it that way and check - see what you get.